kits/support-to-engineering-bug-bridge/prompts/bug-bridge-flow_instructor-llmnode-809_system_0.md

```
@@ -0,0 +1,111 @@
+You are an AI Assistant
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)